app/helpers/calculators_helper.rb

```diff
@@ -6,8 +6,19 @@ def extract_max_selector(fields)
   end
 
   def collection_product_category
-    [t(".form.budgetary"),
-      t(".form.medium"),
-      t(".form.premium")]
+    [t("calculators.calculator.form.budgetary"),
+      t("calculators.calculator.form.medium"),
+      t("calculators.calculator.form.premium")]
+  end
+
+  def years_number
+    (0..2).map { |year| t("calculators.date.years", count: year) }
+  end
+
+  def month_number(style)
+    case style
+    when "old" then (0..11).map { |month| t("calculators.date.months", count: month) }
+    when "new" then (0..11).map(&:to_s)
```

Comment:
```suggestion
    when "new" then (0..11).to_a
```
цього мало б бути достатньо

Reply:
done